### PR TITLE
Refactore AllJoyn samples content

### PR DESCRIPTION
**Pull request by zhuridartem**
As part of fix for https://github.com/ms-iot/samples/issues/64, AllJoyn samples is moving to source form, and AllJoyn Explorer binaries are moved out of samples. Pull request has been submitted: https://github.com/ms-iot/samples/pull/157

This is a corresponding content side change that also needs to be merged.
Note that this is touching existing live content, and that content needs to be updated with the samples changes.

### CODE REVIEW

**Comment by zhuridartem** [comment]
FYI, the /samples/ pull request was merged. AllJoyn related /content/ needs to be also merged, otherwise it is currently in broken state.

**Comment by anthony-ngu** [comment]
Please remove the zip files and pdfs from the commits and try to host them in github releases instead. This bloats our repository greatly with each "release"

https://github.com/blog/1547-release-your-software

File: en-US/win10/samples/ZigBeeAdapterTutorial.md [hunk 1]
```diff
@@ -70,24 +71,15 @@ You can verify that devices have by using “network discovery” feature of XCT
...
-You should see something like that: "_USB\VID_0403&PID_6001\... Name: FT232R USB UART Driver is running_".
-Note that PID could also be 6015 instead of 6001. This depends on the XBee Explorer USB dongle your using. 
- 8. Reboot the RaspBerry Pi2
 
 ## Deploy the ZigBee adapter on your Windows 10 machine
-1. Download the ZigBeeAdapter.zip file [here](https://github.com/ms-iot/samples/blob/develop/AllJoyn/AllJoynZigBeeAdapter/ZigBeeAdapter.zip?raw=true)
-2. Navigate to the folder where you downloaded the zip file. Right click the file and “Extract All…” to the folder of your choice.
-3. Navigate to the extracted folder and open the ZigBeeAdapter.sln solution file in Visual Studio.
-4. Select the relevant target (x86, x64 or ARM) and build the solution in Visual Studio.
+1. Download the AllJoyn Samples code from [here](https://github.com/ms-iot/samples/tree/develop/AllJoyn/){:target="_blank"} to a location on your local machine.
```
Comment (bfjelds) [inline en-US/win10/samples/ZigBeeAdapterTutorial.md]: consider using this style of "download our code":

You can find the source code for this sample by downloading a zip of all of our samples [here](https://github.com/ms-iot/samples/archive/develop.zip) and navigating to the samples-develop\AllJoyn.

**Comment by zhuridartem** [comment]
All comments addressed.

**Pull request merged**